### PR TITLE
Fix docs  of `boxes`, `MultiResolutionDetector`. `apply colormap`, `AugmentationSequential`

### PR DESCRIPTION
**Pull request by johnnv1**
a small fix

### CODE REVIEW

File: kornia/augmentation/container/augment.py [hunk 1]
```diff
@@ -26,28 +26,33 @@ class AugmentationSequential(ImageSequential):
```
Comment (johnnv1) [inline kornia/augmentation/container/augment.py:26]: @shijianjian [the image links of this class (L22 and L24)](https://kornia.readthedocs.io/en/latest/augmentation.container.html#kornia.augmentation.container.AugmentationSequential) are broken, should we remove them or it is an easy fix?

Comment (shijianjian) [reply kornia/augmentation/container/augment.py:26]: emm... somebody removed the images in the tutorial repo...

Comment (shijianjian) [reply kornia/augmentation/container/augment.py:26]: Can you add the sequential example images in this repo directly? Code can be found from https://kornia-tutorials.readthedocs.io/en/latest/data_patch_sequential.html.

**Review by edgarriba — APPROVED** [review]

File: kornia/augmentation/container/augment.py (hunk 1 above)
Comment (johnnv1) [reply kornia/augmentation/container/augment.py:26]: I have added it to the generate images script, just check if these example makes sense:

AugmentationSequential

PatchSequential

Comment (shijianjian) [reply kornia/augmentation/container/augment.py:23]: Great. I think the grid `4x4` can look better for `PatchSequential`.

**Review by shijianjian — APPROVED** [review]

**Pull request merged**